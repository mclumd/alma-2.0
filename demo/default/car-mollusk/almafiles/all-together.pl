rel(cephalopod, steve).
rel(nautilus, nancy).
rel(naked_nautilus, barry).

if(now(6), obs(quote(rel(cephalopod, sam)))).

obs(quote(rel(gas_tank_car, ford))).
obs(quote(not(rel(fueled, ford)))).

obs(quote(rel(electric_car, prius))).
obs(quote(rel(gas_tank_car, prius))).
obs(quote(rel(charged, prius))).
obs(quote(not(rel(fueled, prius)))).
