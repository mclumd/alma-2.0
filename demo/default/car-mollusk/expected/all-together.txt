rel(cephalopod, steve)
rel(mollusk, steve)
abnormal(steve, mollusk, "~rel(shell_bearer, steve)")
~rel(shell_bearer, steve)
rel(nautilus, nancy)
rel(cephalopod, nancy)
rel(mollusk, nancy)
abnormal(nancy, cephalopod, "rel(shell_bearer, nancy)")
rel(shell_bearer, nancy)
rel(naked_nautilus, barry)
rel(nautilus, barry)
rel(cephalopod, barry)
rel(mollusk, barry)
abnormal(barry, mollusk, "~rel(shell_bearer, barry)")
abnormal(barry, nautilus, "~rel(shell_bearer, barry)")
~rel(shell_bearer, barry)
rel(cephalopod, sam)
rel(mollusk, sam)
abnormal(sam, mollusk, "~rel(shell_bearer, sam)")
~rel(shell_bearer, sam)
rel(car, ford)
rel(gas_tank_car, ford)
~rel(powered, ford)
~rel(fueled, ford)
abnormal(ford, car, "~rel(powered, ford)")
rel(car, prius)
rel(gas_tank_car, prius)
rel(electric_car, prius)
rel(charged, prius)
~rel(fueled, prius)
rel(powered, prius)
abnormal(prius, gas_tank_car, "rel(powered, prius)")
